client/components/gridicon/gridicons.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module 'gridicons/dist/util/icons-offset' {
+	const iconsThatNeedOffset: [string];
+	const iconsThatNeedOffsetX: [string];
+	const iconsThatNeedOffsetY: [string];
```

Comment:
Is a singleton array? Should it be `string[]`?

Reply:
!!

Yes it should. Don't know why I typed that 😕